test/DocumentFormat.OpenXml.Tests/Spreadsheet/CellTests.cs

```diff
@@ -19,6 +19,8 @@ public class CellTests
         [InlineData("false", CellValues.Boolean, true)]
         [InlineData("true", CellValues.Boolean, true)]
         [InlineData("other", CellValues.Boolean, false)]
+        [InlineData("0", CellValues.Boolean, true)]
```

Comment:
Should it provide a `false` result instead?

Reply:
These tests are actually checking if validation flags the contents or not. The `true` here means that it is a valid value for a boolean